common/src/main/java/org/schemarepo/config/Config.java

```diff
@@ -90,6 +105,9 @@
 
     DEFAULTS.setProperty(DEFAULT_SUBJECT_VALIDATORS,"");
 
+    // Validation defaults. The read strategy class is "Always Fail" unless configured.
+    DEFAULTS.setProperty(VALIDATION_READ_STRATEGY_CLASS, "org.schemarepo.validation.AlwaysFailValidationStrategy");
```

Comment:
[re-posting this answer here, to preserve the thread]

I totally see where you're coming from and I think it's valuable to maintain the same functionality you described in your example config.

I'm just trying to think about what could be the bare minimum config needed to get up and running, which would require as little knowledge as possible about the implementation details of Guice and the schema-repo.